mmcls/models/backbones/regnet.py

```diff
@@ -45,23 +45,32 @@ class RegNet(ResNet):
     Example:
         >>> from mmcls.models import RegNet
         >>> import torch
-        >>> self = RegNet(
-                arch=dict(
-                    w0=88,
-                    wa=26.31,
-                    wm=2.25,
-                    group_w=48,
-                    depth=25,
-                    bot_mul=1.0))
-        >>> self.eval()
         >>> inputs = torch.rand(1, 3, 32, 32)
-        >>> level_outputs = self.forward(inputs)
+        >>> # use str type 'arch'
+        >>> # Note that default out_indices is (3,)
+        >>> regnet_cfg = dict(arch='regnetx_4.0gf')
+        >>> model = RegNet(**regnet_cfg)
+        >>> model.eval()
+        >>> level_outputs = model(inputs)
         >>> for level_out in level_outputs:
         ...     print(tuple(level_out.shape))
-        (1, 96, 8, 8)
-        (1, 192, 4, 4)
-        (1, 432, 2, 2)
-        (1, 1008, 1, 1)
+        ...
+            (1, 1360, 1, 1)
```

Comment:
I am sorry for  later reply.
There are still some errors.
![image](https://user-images.githubusercontent.com/18586273/183325767-6365db36-4c4a-4cf3-b769-058097f3af29.png)

please remove the ``·blank line```

you can check the update in the link
![image](https://user-images.githubusercontent.com/18586273/183326209-9c3748f3-0375-48d5-9793-731637c7e92f.png)